bom/application/pom.xml

```diff
@@ -32,7 +32,7 @@
         <jaeger.version>1.4.0</jaeger.version>
         <quarkus-http.version>4.1.1</quarkus-http.version>
         <jboss-servlet-api_4.0_spec.version>1.0.0.Final</jboss-servlet-api_4.0_spec.version>
-        <micrometer.version>1.7.0</micrometer.version> <!-- when updating, verify if com.google.auth should not be updated too -->
```

Comment:
I'm surprised you removed the comment? We do not need google-auth to be aligned anymore?

Reply:
Nope. The google stuff was related to stackdriver, which moved out to quarkiverse